control_panel_api/k8s.py

```diff
@@ -0,0 +1,60 @@
+from django.http import HttpResponse
+from kubernetes import client, config
+from kubernetes.config.config_exception import ConfigException
+import requests
+
+
+def proxy(request):
+    return Request(request).make()
+
+
+class Request(object):
+
+    def __init__(self, request):
+        self.request = request
+        self._config = Config()
+
+    def make(self):
```

Comment:
A better name for this method might be `dispatch`

Reply:
I called it `make` because usually you usually _make_ requests, but I'll change 👍

Reply:
Yeah, I get that, but `make` is a bit unspecific - usually it would mean "build"